src/tasks/Microsoft.NET.Sdk.WebAssembly.Pack.Tasks/AssetsComputingHelper.cs

```diff
@@ -64,6 +64,7 @@ public class AssetsComputingHelper
             ".dat" when invariantGlobalization && fileName.StartsWith("icudt") => "invariant globalization is enabled",
             ".dat" when loadFullICUData && fileName != "icudt" => "full ICU data is enabled",
             ".dat" when hybridGlobalization && fileName != "icudt_hybrid" => "hybrid globalization is enabled",
+            ".json" when !hybridGlobalization && fileName == "segmentation-rules" => "segmentation-rules.json is only used when hybrid globalization is enabled",
```

Comment:
This is correct but could we invert the conditions to match the convention of other lines? It requires less thinking when reading.

Reply:
I had to revert the change back to `".json" when !hybridGlobalization && fileName == "segmentation-rules"` because we need to filter-out the `segmentation-rules.json` file when HG is not enabled. We could add both versions to make sure the segmentation rules have the correct name, but I don't think that is necessary because we specifically include this file so we should be ok https://github.com/dotnet/runtime/blob/0199ab62f2a547cf5b139f0dfbd152d6519d6773/src/mono/browser/browser.proj#L361.